pynest/nest/lib/hl_api_spatial.py

```diff
@@ -810,6 +888,87 @@ def GetTargetPositions(sources, tgt_layer, syn_model=None):
     return [src_tgt_pos_map[snode_id] for snode_id in sources.tolist()]
 
 
+def GetSourcePositions(src_layer, targets, syn_model=None):
+    """
+    Obtain positions of sources to a given `NodeCollection` of `targets`.
+
+    For each neuron in `targets`, this function finds all source elements
+    in `src_layer`. If `syn_model` is not given (default), all targets are
+    returned, otherwise only source of specific type.
```

Comment:
```suggestion
    returned, otherwise only sources connected via the given synapse model.
```
Please check corresponding methods and make the corresponding change.

Reply:
Done in c034c34